pkg/installmanager/actuator.go

```diff
@@ -0,0 +1,16 @@
+package installmanager
+
+import (
+	hivev1 "github.com/openshift/hive/pkg/apis/hive/v1"
+	log "github.com/sirupsen/logrus"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Actuator interface is the interface that is used to add cloud provider support to the deprovision controller.
+type Actuator interface {
+	// CanHandle returns true if the actuator can handle a particular case
+	CanHandle() bool
```

Comment:
Suggest: IsConfigured(), CanHandle sounds like we'd provide it an argument to see if it can handle that, but that is not our case here.